concepts.md

```diff
@@ -368,6 +368,8 @@ command line line tools) is as follows.
 
 1. Load input object.
 1. Load, process and validate a CWL document, yielding one or more process objects.
+The [`$namespaces`](SchemaSalad.html#Explicit_context) present in the CWL document
```

Comment:
Maybe tweak this to say the $namespaces specifically come from the top level document